src/include/ipc4/copier.h

```diff
@@ -70,16 +70,6 @@ struct ipc4_base_module_cfg_ext {
 	struct ipc4_output_pin_format output_pins[1];
 } __attribute__((packed, aligned(4)));
 
-/* This enum defines short 16bit parameters common for all modules.
- * Value of module specific parameters have to be less than 0x3000.
- */
-enum ipc4_base_module_params {
-	/* handled inside LargeConfigGet of module instance */
-	IPC4_MOD_INST_PROPS  = 0xFE,
-	/* handled inside ConfigSet of module instance */
-	IPC4_MOD_INST_ENABLE = 0x3000
-};
-
```

Comment:
@mwasko @abonislawski @aborisovich @marcinszkudlinski Any plans to use these, or do we go ahead and remove?

Reply:
Support for MOD_INST_ENABLE is required for IADK module adapter API.
Support for MOD_INST_PROPS is required for both: Module API and IADK module adapter API.

It seems in SOF this implementation is missing but it is required and used at least by Win driver.
I think it should be moved to generic module header, but not removed.

Reply:
@wszypelt do we test MOD_INST_PROPS for closed-source ACE FW? do we test it for SOF? I can see the test name is check_mod_inst_props_get() in fw_02_base_fw_api/test_00_debug_api

Reply:
how about ipc4/base-config.h, the same as you do with ipc4_base_module_cfg_ext ?

Reply:
OK, then I might just use one patch to do these.

Reply:
@ujfalusi Thank you!